### PR TITLE
fix: broken page issue

### LINKED ISSUE #1142: 🐛 Bug: Blog page Table of Content is Broken
### Describe the bug

Table of contents (on this page section) is found broken for the [blog ](https://json-schema.org/blog/posts/applicability-json-schema-fundamentals-part-1)

While checking for other pages its found that mentioned component is working as expected  

### Steps To Reproduce

1. Visit https://json-schema.org/blog/posts/applicability-json-schema-fundamentals-part-1
2. Broken component can be seen there

### Expected Behavior

It should not be broken and must be fixed maintaining the consistency with other blog pages

### Screenshots

image

### Device Information [optional]

```markdown
- OS: Linux (Debian)
- Browser: Firefox
- version: Latest
```

### Are you working on this issue?

No

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No

### PR DESCRIPTION
**Pull request by arpitkuriyal**
**What kind of change does this PR introduce?**
CSS correction in the Table section of this [blog](https://json-schema.org/blog/posts/applicability-json-schema-fundamentals-part-1) page

**Issue Number:**

-  Closes #1142 

___ 

**Screenshots/videos:**
Screenshot 2024-11-28 at 11 06 40 PM

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

### CODE REVIEW

**Comment by Relequestual** [comment]
This does fix the style issue, but any idea why it's different from other posts?

**Comment by Relequestual** [comment]
Thanks for your work too of course =]

File: _includes/community
Comment (Relequestual) [inline _includes/community:1]: I think it best this is part of a separate PR

**Comment by DhairyaMajmudar** [comment]
@arpitkuriyal in this page its expected not to have bullet points (The blue dots)

Just below image 

image

**Comment by arpitkuriyal** [comment]
@DhairyaMajmudar  okay i will remove them.

File: _includes/community
Comment (arpitkuriyal) [reply _includes/community:1]: @Relequestual  i also get little confused while doing it as it is different from the other posts.  i will correct it and make a new PR soon

**Comment by arpitkuriyal** [comment]
now it is like this
Screenshot 2024-11-29 at 3 46 49 PM

**Comment by benjagm** [comment]
This has been already resolved. Thanks for your contributions.

**Pull request closed without merging**